bindings/matrix-sdk-ffi/src/client_builder.rs

```diff
@@ -27,6 +27,44 @@ enum HomeserverConfig {
     ServerNameOrUrl(String),
 }
 
+#[derive(uniffi::Enum)]
+pub enum BackupDownloadStrategy {
```

Comment:
nit: instead of copying the enum, can you add the derive with a `feature = 'uniffi'` guard to the original enum?

Reply:
Much better, done in 71c6878697 !